nativelib/src/main/scala/scala/scalanative/posix/pthread.scala

```diff
@@ -72,6 +75,9 @@ object pthread {
                             stackaddr: Ptr[Byte],
                             stacksize: CSize): CInt = extern
 
+  def pthread_attr_setstackaddr(attr: Ptr[pthread_attr_t],
+                                stackaddr: Ptr[Byte]): CInt = extern
+
   def pthread_attr_setstacksize(attr: Ptr[pthread_attr_t],
                                 stacksize: CSize): CInt = extern
 
```

Comment:
Looks like all the `pthread_barrier_*` stuff below here are also missing. The standard labels it as "ADVANCED REALTIME THREADS", so not required for "simple" multithreading.